activesupport/lib/active_support/actionable_error.rb

```diff
@@ -0,0 +1,52 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module ActiveSupport
+  # Actionable errors let's you define actions to resolve an error.
+  #
+  # To make an error actionable, include the <tt>ActiveSupport::ActionableError</tt>
+  # module and invoke the +action+ class macro to define the action. An action
+  # needs a name and a block to execute.
+  module ActionableError
+    extend Concern
+
+    NonActionable = Class.new(StandardError)
```

Comment:
I thought we used the `class NonActionable < StandardError; end` style?

Reply:
Okay, will change this.